metricbeat/module/docker/diskio/helper.go

```diff
@@ -0,0 +1,106 @@
+package diskio
+
+import (
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/metricbeat/module/docker"
+	dc "github.com/fsouza/go-dockerclient"
```

Comment:
Only a minor comment: an internal convention we have is that a new line is put between golang imports, then external imports, and local imports. So the above would be:

```
import (
	"time"

	dc "github.com/fsouza/go-dockerclient"
	
	"github.com/elastic/beats/libbeat/logp"
	"github.com/elastic/beats/metricbeat/module/docker"
)
```